### PR TITLE
HToAATo4B production cards: VBFH

### PR DESCRIPTION
**Pull request by abrinke1**
Generation script, template cards, and cards for all mass points for VBFH.
See https://github.com/cms-sw/genproductions/pull/2880

### CODE REVIEW

**Comment by abrinke1** [comment]
Hi @sihyunjeon , @agrohsje ,

I have updated the VBF production with the fixes from ttH (https://github.com/cms-sw/genproductions/pull/2903).  The remaining differences in the ggH vs. ttH run_card are the following:

ickkw = 1  (instead of 0)
auto_ptj_mjj = T  (instead of F)
ptj = 20  (instead of 0)
xqcut = 20  (instead of 0)

I assume ickkw is necessary since the proc_card includes a process with two extra jets.  I'm not sure whether the other settings are correct, or if I need extra modifications for VBF: please let me know.

Thanks much,
@abrinke1

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat:65]: You don't need to turn on matchings. You don't have multipartons.
it is only used for the case where you do
p p > X
p p > X j

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat:81]: turn this off

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat (hunk 1 above)
Comment (abrinke1) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat:65]: Hi @sihyunjeon ,
Does this line where I have:
p p > h01 j j
not count as "multipartons"?  Is it only if I have two different lines, with different numbers of extra jets "j"?
https://github.com/abrinke1/genproductions/blob/HToAATo4B_Cards_Run2_UL_VBFH/bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_proc_card.dat#L5

Comment (sihyunjeon) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat:65]: Definition of multiparton is when you need jet matching.
So "p p > X j j" alone is not multiparton sample
if you also have "p p > X j j j", it is

**Comment by agrohsje** [comment]
Hi @sihyunjeon feel free to merge this PR once the ickkw and auto_ptj_mjj are fixed. @abrinke1 please follow up so we can merge this asap.

**Comment by abrinke1** [comment]
Hi @agrohsje , @sihyunjeon ,
Sorry for the delay: the run_card has been fixed now.
Best regards,
@abrinke1

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 20   = xqcut   ! minimum kt jet measure between partons
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat:259]: This should be 0 as well (it's jet matching parameter)

Comment (abrinke1) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_VBFH_HToAATo4B_M-MASS/SUSY_VBFH_HToAATo4B_M-MASS_run_card.dat]: Hi @sihyunjeon ,
Thanks for the catch!  Fixed now.
Best, @abrinke1

**Comment by abrinke1** [comment]
Hi @sihyunjeon , @agrohsje ,
Any further comments or questions on the VBF cards?
Thank, @abrinke1

**Pull request merged**